tools/wpt/run.py

```diff
@@ -493,6 +495,26 @@ def setup_wptrunner(venv, prompt=True, install_browser=False, **kwargs):
     setup_cls = product_setup[kwargs["product"]](venv, prompt, sub_product)
     setup_cls.install_requirements()
 
+    if kwargs["affected"]:
```

Comment:
`if kwargs.pop("affected")`

Reply:
Rather `revish = kwargs.pop("affected")` and `if revish` I guess?